src/templates/learn-doc.js

```diff
@@ -35,30 +35,43 @@ const ContentRow = ({ children }) => (
 );
 
 const getChildren = (path, navLinks) => {
-  return navLinks.filter(
-    (node) =>
-      node.fields.path.includes(path) &&
-      node.fields.path.split('/').length === path.split('/').length + 1,
-  );
+  return navLinks
+    .filter(
+      (node) =>
+        node.fields.path.includes(path) &&
+        node.fields.path.split('/').length === path.split('/').length + 1,
+    )
+    .sort((a, b) => a.fields.path.localeCompare(b.fields.path));
 };
 
-const Tiles = ({ mdx, navLinks }) => {
+const Tiles = ({ mode, mdx, navLinks }) => {
+  const modes = {
+    None: 'none',
+    Simple: 'simple',
+    Full: 'full',
+  };
+  if (mode === modes.None) return null;
+
   const { path } = mdx.fields;
-  const depth = path.split('/').length;
-  if (depth === 4) {
+
+  if (!mode) {
+    const depth = path.split('/').length;
+    if (depth === 4) mode = modes.Full;
+    else if (depth >= 5) mode = modes.Simple;
+  }
+
+  if (Object.values(modes).includes(mode)) {
+    const colSize = mode === 'simple' ? 4 : 6;
```

Comment:
Maybe it makes sense to store these widths with the mode names? Maybe the default depth stuff too. Something like ...

`Full: { name: 'full', defaultDepth: 4, columnWidth: 6 }`

Just to keep the little bits "configuration" closer together. What do you think?

Reply:
Moved into CardDecks - this was a weird option to push up, only reason I can think of to change it is if design of the cards themselves change.